examples/client/Locomotion/src/pages/AuthScreens/Onboarding/styles.js

```diff
@@ -57,6 +57,7 @@ export const Name = styled(BaseText)`
 ${FONT_SIZES.H1};
 ${FONT_WEIGHTS.BOLD};
 margin: 20px;
+max-width: 55%;
```

Comment:
[Check if you can add elip](https://stackoverflow.com/questions/30594080/how-to-have-ellipsis-effect-on-text)